### PR TITLE
Pymol tcl tk

### PR DESCRIPTION
**Pull request by romainstuder**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by romainstuder** [comment]
Tcl/Tk stable release 8.6.7

File: Formula/tcl-tk.rb [hunk 1]
```diff
@@ -1,19 +1,19 @@
 class TclTk < Formula
   desc "Tool Command Language"
   homepage "https://www.tcl.tk/"
-  revision 2
+  revision 1
```
Comment (fxcoudert) [inline Formula/tcl-tk.rb]: Please remove the `revision` line.

File: Formula/tcl-tk.rb [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
   stable do
```
Comment (fxcoudert) [inline Formula/tcl-tk.rb]: Remove the `stable` block

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Review by romainstuder — COMMENTED** [review]
Revision line removed. "Stable" removed.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @romainstuder for the pull request!

**Pull request closed without merging**